typescript/packages/lookslike-high-level/src/recipes/reading-list.tsx

```diff
@@ -2,27 +2,26 @@ import { h } from "@commontools/html";
 import { recipe, handler, UI, NAME, derive, lift, cell } from "@commontools/builder";
 import { z } from "zod";
 
-const BookItem = z.object({
+const ArticleItem = z.object({
   title: z.string(),
-  author: z.string(),
-  done: z.boolean().default(false),
+  url: z.string(),
 });
```

Comment:
Similar, just so I can play with data shapes, can revert if someone is fond of the old one.

Reply:
good with me!